runtime/statemine/src/lib.rs

```diff
@@ -323,16 +324,22 @@ parameter_types! {
 	// https://github.com/paritytech/substrate/blob/069917b/frame/assets/src/lib.rs#L257L271
 	pub const MetadataDepositBase: Balance = deposit(1, 68);
 	pub const MetadataDepositPerByte: Balance = deposit(0, 1);
+	pub const UnitBody: BodyId = BodyId::Unit;
```

Comment:
I think this was set to `Unit` in the context of Rococo, but on Polkadot/Kusama, we want it to be the Council.
```suggestion
	pub const UnitBody: BodyId = BodyId::Executive;
```

Reply:
I think that `Unit` this is still valid for Council. In any case I want to suggest to make the change in a follow up PR as the Aura PR will bring many changes and I want to merge this.

Reply:
Should be Executive. Unit is not valid.